tests/ActivityTest.php

```diff
@@ -43,26 +43,17 @@ public function testInstantiation() {
     }
 
     public function testFromJSONInvalidNull() {
-        $this->setExpectedException(
-            'InvalidArgumentException',
-            'Invalid JSON: ' . JSON_ERROR_NONE
-        );
+        $this->setExpectedException('TinCan\JSONParseErrorException');
```

Comment:
Yep, I worked through that, and then determined that getting to a point where it was supported in PHPUnit meant we had to drop support for 5.5 which I wasn't ready to do (and still aren't even though I appreciate the comment about 5.5 being EOLd). The fact that it was *just* EOLd and that it is still the version shipped with Mac OS X means I don't think we have the luxury of dropping both 5.4 and 5.5 in the same release. The switch from `setExpectedException` to `expectException` is straight forward when we have to make it, and I don't think PHPUnit will be able to actually drop it that quickly, being deprecated just means we need to be prepared and have a path forward, I think we are good there. Particularly since it is just a dev dependency and I don't mind doing a change to just the test files as part of a minor (or patch) release.